packages/playback-core/src/index.ts

```diff
@@ -74,9 +87,7 @@ declare global {
   }
 }
 
-export type HTMLMediaElementProps = Partial<
-  Pick<HTMLMediaElement, "src" | "autoplay">
```

Comment:
`autoplay` isn't needed here, from what I can tell.

Reply:
hmm... this may be one of the aforementioned relationships/definitional threads. I'm cool with removing it if you confirmed that this won't cause issues of e.g. typescript + `mux-video` _*usage*_ of the autoplay _*prop*_. Given your updated definitions above, I think we should be fine though (I can help verify).

Reply:
Yeah, I think it isn't necessary, but I'll double-check tomorrow. If I set `autoplay` on `mux-video` in one of the demos, I don't think it complains about this.

Reply:
yeah it's a "prop" vs. "attribute" case here, so you'd need to confirm in the TypeScript/JS use case of the element instance, but I think we should be fine (🤞)